recipes/sentry-native/all/conanfile.py

```diff
@@ -30,42 +30,47 @@ class SentryNativeConan(ConanFile):
         "transport": "curl"
     }
 
+    _cmake = None
+
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
-    _cmake = None
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+        if self.options.backend == "inproc" and self.settings.os == "Windows" and tools.Version(self.version) < "0.4":
+            raise ConanInvalidConfiguration("The in-process backend is not supported on Windows")
+        if self.options.transport == "winhttp" and self.settings.os != "Windows":
+            raise ConanInvalidConfiguration("The winhttp transport is only supported on Windows")
 
     def requirements(self):
         if self.options.transport == "curl":
-            self.requires("libcurl/7.71.0")
-
+            self.requires("libcurl/7.74.0")
         if self.options.backend == "crashpad":
             raise ConanInvalidConfiguration("crashpad not available yet in CCI")
-        if self.options.backend == "breakpad":
+        elif self.options.backend == "breakpad":
             raise ConanInvalidConfiguration("breakpad not available yet in CCI")
```

Comment:
These 2 options values should be removed from the possible set (with a fixme comment) if they are not usable

Reply:
It was already there before this PR, not sure that removing those values now is a good idea. Others recipes use this pattern when recipes are missing in CCI.